### PR TITLE
bin/xbps-checkvers: list removed templates

### PR DESCRIPTION
**Pull request by Chocimier**
Step towards #76.

### CODE REVIEW

File: bin/xbps-checkvers/main.c [hunk 1]
```diff
@@ -564,7 +571,12 @@ rcv_check_version(rcv_t *rcv)
 	reverts = NULL;
 	xbps_dictionary_get_cstring_nocopy(rcv->env, "reverts", &reverts);
 
-	sz = snprintf(srcver, sizeof srcver, "%s_%s", version, revision);
+	if (rcv->removed)
+	{
```
Comment (xtraeme) [inline bin/xbps-checkvers/main.c]: these parens are useless and bad indented

File: bin/xbps-checkvers/main.c [hunk 2]
```diff
@@ -89,6 +92,7 @@ show_usage(const char *prog)
 " -D --distdir <dir>	Set (or override) the path to void-packages\n"
 "  			(defaults to ~/void-packages).\n"
 " -d --debug 		Enable debug output to stderr.\n"
+" -e --removed 		List packages present in repos, but not in distdir.\n"
```
Comment (xtraeme) [inline bin/xbps-checkvers/main.c]: Please fix indentation

**Comment by Chocimier** [comment]
Changed according to comments.

**Comment by Chocimier** [comment]
@Duncaen ?

**Comment by xtraeme** [comment]
@Duncaen ping

**Comment by xtraeme** [comment]
@Chocimier please rebase and fix the conflicts

**Comment by Duncaen** [comment]
Does it make sense to do both at once, or should `-e` just list removed templates?

**Comment by xtraeme** [comment]
right now, if srcver cannot be fetched on srcpkgs it prints `?`

I prefer to have `-e` list removed templates, it's clearer

**Comment by Chocimier** [comment]
Changed as requested.

**Pull request merged**